File: backends/php-laravel/app/UseCases/Invitation/Dtos/InvitationDto.php
<?php

/**
 * This is a program developed by BobTabo.
 *
 * Copyright (c) 2026 BobTabo. All Rights Reserved.
 */

declare(strict_types=1);

namespace App\UseCases\Invitation\Dtos;

use App\Support\Dtos\AbstractDto;

/**
 * 招待 API 用の入力 DTO です。
 *
 * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
 * @package App\UseCases\Invitation\Dtos
 */
class InvitationDto extends AbstractDto
{
    public ?string $token = null;
}

File: backends/java-spring/src/main/java/com/authorization/usecases/invitation/dtos/InvitationDto.java
/**
 * This is a program developed by BobTabo.
 *
 * Copyright (c) 2026 BobTabo. All Rights Reserved.
 */
package com.authorization.usecases.invitation.dtos;

import com.authorization.support.dtos.AbstractDto;
import lombok.Getter;
import lombok.Setter;

/**
 * 招待DTOクラスです。
 *
 * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
 */
@Getter
@Setter
public class InvitationDto extends AbstractDto {

    private String token;
    private Integer role;
}
